daemon/src/maker_inc_connections.rs

```diff
@@ -102,6 +97,11 @@ pub struct Actor {
     settlement_actors: AddressMap<OrderId, collab_settlement_maker::Actor>,
     rollover_actors: AddressMap<OrderId, rollover_maker::Actor>,
     tasks: Tasks,
+
+    /// Tracks the last order that we sent out to our connected takers.
+    ///
+    /// Storing this here allows us to send it to newly connected takers.
+    last_order: Option<Order>,
```

Comment:
🤔 This seems okay for now, but should we consider creating a little `order::Actor` which takes care of this state? It seems like the order doesn't belong in the "connection" actor.

Reply:
But then we have to send requests back and forth again which lead to a deadlock in the previous version. I am unsure!

Reply:
I get your point and there's no need to change this with this PR anyway. Some thoughts (from now on I will talk about offers, since I think that's the right term we agreed on):

If the maker's `offer::Actor` can only receive a new offer from the HTTP API and respond to a `GetOffer` request from the `maker_inc_connections::Actor`, is there potential for a deadlock?

Reply:
I think there can be if there is a circular dependency in general.

Reply:
Do you see a request with a direct response (in the context of `xtra`, a `Message` with a `Result` other than `()`) as a circular dependency?

Reply:
If both actors send messages to each other, that is a circular dependency.

Reply:
Perhaps a dedicated `offer::Actor` would not create a circular dependency, I haven't fully thought it through :)